### PR TITLE
[player-1256]

### PR DESCRIPTION
**Pull request by aeng7**
-updated constants to include SDK_AD_EVENT

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: js/utils/constants.js [hunk 1]
```diff
@@ -1068,6 +1068,15 @@
        */
       SHOW_AD_MARQUEE: "showAdMarquee",
 
+      /**
+       * An ad plugin will publish this event whenever the ad SDK throws an ad event. Typical ad evernts are
```
Comment (jpfranco) [inline js/utils/constants.js]: evernts :p

Comment (pilievOoyala) [inline js/utils/constants.js]: gerss  bermps mah fervret berks!

Comment (pilievOoyala) [inline js/utils/constants.js]: image

Comment (aeng7) [inline js/utils/constants.js]: Screw it, I'm gonna live life on the erdge

Comment (aeng7) [inline js/utils/constants.js]: I'll have to fix this in mjolnir first if we want to fix it.

Comment (jpfranco) [inline js/utils/constants.js]: Probably not worth another PR considering that it's private

**Pull request merged**